doc/doxygen/src/MainPage.dox

```diff
@@ -1,5 +1,27 @@
 /*! @mainpage An Introduction
  *
+ * @note Looking for resources?
+ * * 🐍 Get OpenAssetIO from <a
+     href="https://pypi.org/project/openassetio/">PyPI</a>
+ * * 🛠️ Learn how to <a
+     href="https://github.com/OpenAssetIO/OpenAssetIO/blob/main/doc/BUILDING.md">build
+     OpenAssetIO</a>
+ * * 📐 Get off to a good start with the <a
+     href="https://github.com/OpenAssetIO/Template-OpenAssetIO-Manager-Python">python
+     manager template.</a>
+ * * 📖 See some examples: <a
+     href="https://github.com/ynput/ayon-openassetio-manager-plugin/tree/main#readme">Ayon
+     manager plugin</a>, <a
+     href="https://github.com/OpenAssetIO/usdOpenAssetIOResolver#readme">USD
+     Asset Resolver prototype</a>, <a
+     href="https://github.com/OpenAssetIO/otio-openassetio">OTIO media
+     linker</a>
+ * * 📇 Check out the <a
+     href="https://github.com/OpenAssetIO/OpenAssetIO/">OpenAssetIO
+     codebase</a>
+ * * 📖 Or find some additional examples in <a
+     href="https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/tree/main/examples/">OpenAssetIO-MediaCreation</a>
```

Comment:
[consider] Perhaps worth linking first specifically to the  "Hello OpenAssetIO" notebook, then a second link to the directory of notebooks (also worth noting they're not just examples but documentation, perhaps use the word "tutorial"?)
```suggestion
 * * 📖 See a <a
     href="https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/blob/main/examples/hello_openassetio.ipynb">Hello World</a> tutorial example, with more tutorial examples to be found in <a
     href="https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/tree/main/examples/">OpenAssetIO-MediaCreation</a>
```

Reply:
I'm very slightly uneasy about the word tutorial for the notebooks, as they're not _quite_ what I would call full tutorials, more like tutorialettes :P. Took the wording nonetheless though, it's fine.